android/app/src/main/res/layout/launch_screen.xml

```diff
@@ -0,0 +1,21 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+>
+
+    <EditText
```

Comment:
used text field instead of edit text field.

Reply:
Done